packages/odf/components/kms-config/kms-config.tsx

```diff
@@ -74,10 +82,26 @@ export const KMSConfigure: React.FC<KMSConfigureProps> = ({
         type: 'securityAndNetwork/setKmsProvider',
         payload: provider,
       });
+      setIsOpen(false);
     },
     [dispatch]
   );
 
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const toggle = (toggleRef: React.Ref<MenuToggleElement>) => (
+    <MenuToggle
+      ref={toggleRef}
+      onClick={onToggleClick}
+      isExpanded={isOpen}
+      className={'pf-v5-c-form-control'}
```

Comment:
u really need this class ??
```suggestion
      className='pf-v5-c-form-control'
```

Reply:
yes, otherwise the size is very small and not align with other text box in the ui